Makefile

```
@@ -0,0 +1,58 @@
+export PYTHONPATH := $(CURDIR)/lib:$(CURDIR)/test
+addon_xml := addon.xml
+
+# Collect information to build as sensible package name
+name = $(shell xmllint --xpath 'string(/addon/@id)' $(addon_xml))
+version = $(shell xmllint --xpath 'string(/addon/@version)' $(addon_xml))
+git_branch = $(shell git rev-parse --abbrev-ref HEAD)
+git_hash = $(shell git rev-parse --short HEAD)
+
+zip_name = $(name)-$(version)-$(git_branch)-$(git_hash).zip
```

```suggestion
zip_name = $(name)-$(version)-$(subst /,_,$(git_branch))-$(git_hash).zip
```

It took me a bit to get my environment all set up for this everything to work here, but it did fail here because I checkout out your PR directly such that the branch name was `pr/8/merge`. This change fixes that case.

---

Sure, go ahead. It as indeed designed for branches without slashes.